client/src/components/ProfileEdit.jsx

```diff
@@ -7,94 +7,97 @@ import { Link } from 'react-router-dom'
 class ProfileEdit extends Component {
 
     state = {
-        user_edit: {},
-        error_nick: "",
-        error_repeated: "",
-        error_first: "",
-        error_last: "",
+        user: {},
+        errors_edit: {}
     }
 
     componentDidMount() {
         request('/profile/' + this.props.match.params.profile_id)
         .then(json => {
-            this.setState({user_edit: json});
+            this.setState({user: json});
             this.setState({
-                user_edit: {
-                    f_name: this.state.user_edit.first_name,
-                    l_name: this.state.user_edit.last_name,
-                    nick: this.state.user_edit.nickname,
-                    loc: this.state.user_edit.location,
-                    pass: this.state.user_edit.password
+                user: {
+                    first_name_input: this.state.user.first_name,
+                    last_name_input: this.state.user.last_name,
+                    nickname_input: this.state.user.nickname,
+                    location_input: this.state.user.location,
```

Comment:
Have you any reasons to rename this states?

Reply:
Please, don't ignore me, just give me the answer.

Reply:
Well, I'm sorry for being a bit busy at the time, I renamed those states to better reflect what they are actually doing, and was that too hard to figure it out yourself?

Reply:
I think it not very important to add `_input` to each state, also it will be easier to understand without it. You have user obj in a state but it uses so rarely. It is just my opinion, and all I want to know, have you any reasons to rename it or just "better reflect"?